src/AppBundle/Controller/EventManagement/EventController.php

```diff
@@ -0,0 +1,152 @@
+#            ->setMethod("get")
+#            ->getForm();
+#
+#        $filter = new Filter();
+#
+#        //Handel the form input
+#        $userSearchForm->handleRequest($request);
+#        if ($userSearchForm->isSubmitted() && $userSearchForm->isValid()) {
+#            $data = $userSearchForm->getData();
+#            $filter->addFilter($data["filterOption"], $data["filterText"]);
+#        }
+
+        $repository = $this->getDoctrine()->getRepository(Event::class);
+        $events = $repository->findAll();
+
+        return $this->render('eventManagement/showAllEvents.html.twig', [
+#            "peopleSearchForm" => $userSearchForm->createView(),
+            "events"=>$events,
+        ]);
+    }
+
+
+
+    /**
+     * @Route("/events/add", name="addEvent")
+     * @Security("has_role('ROLE_stavo')")
+     * @param Request $request
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function addEvent(Request $request)
+    {
+        // you can fetch the EntityManager via $this->getDoctrine()
+        // or you can add an argument to your action: createAction(EntityManagerInterface $em)
+        $em = $this->getDoctrine()->getManager();
+
+        $event = new Event();
+        /*
+        $event->setName("Test");
+        $event->setDescription("qwertzu qwertz qwertzu");
+        $event->setPriceInCent(rand(1,200));
+        $date = new DateTime(rand(2000,2020).'-01-01');
+        $event->setDateFrom($date);
+        $event->setDateTo($date);
+        $event->setPlace("at home");
+
+        // tells Doctrine you want to (eventually) save the Product (no queries yet)
+        $em->persist($event);
+
+        // actually executes the queries (i.e. the INSERT query)
+        $em->flush();
+
+        $this->addFlash("success", "Saved new random event with id ".$event->getId());
+        return $this->redirectToRoute("allEvents");
+        */
+
+
+        $addAnEventForm = $this->createFormBuilder($event)
+            ->add('Name', TextType::class)
+            ->add('Description',TextareaType::class)
+            ->add('PriceInCent',IntegerType::class)
+            ->add('DateFrom', DateTimeType::class)
+            ->add('DateTo', DateTimeType::class)
+            ->add('Place',TextareaType::class)
+            ->add('save', SubmitType::class, array('label' => 'Erstellen'))
+            ->getForm();
+
+        $addAnEventForm->handleRequest($request);
+
+        if ($addAnEventForm->isSubmitted() && $addAnEventForm->isValid()) {
+            // $form->getData() holds the submitted values
+            // but, the original `$task` variable has also been updated
+            $event_data = $addAnEventForm->getData();
+
+            // ... perform some action, such as saving the task to the database
+            // for example, if Task is a Doctrine entity, save it!
+            $em = $this->getDoctrine()->getManager();
+            $em->persist($event_data);
```

Comment:
Was passiert, wenn das Event schon existiert?

Reply:
Was meinst du mit "das Event schon existiert"? Gleicher Event-Name -> neue MySQL Id

Reply:
Ja genau, aber warum sollte es ein Event mit dem selben Namen geben? Darum würde ich überprüfen, ob der Name schon existiert. Wie soll der User sonst in der Tabelle die zwei Events unterscheiden?